trello.py

```diff
@@ -0,0 +1,64 @@
+from item import Item
+from flask_config import Config
+import requests
+
+trello_board_url = 'https://api.trello.com/1/boards/5ef0aa880a06bf1a91b44992/cards'
+trello_cards_url = 'https://api.trello.com/1/cards'
+todo_list_id = '5ef0aa880a06bf1a91b44993'
+done_list_id = '5ef0aa880a06bf1a91b44995'
+
+auth_params = { 'key': Config.TRELLO_API_KEY,'token': Config.TRELLO_API_TOKEN }
+
+def get_trello_card_url(id):
+    return f'{trello_cards_url}/{id}'
+
+def map_card_to_item(card):
```

Comment:
This method could be moved into the `Item` class (hint: try using `@classmethod`)